config/intellij-idea-inspections.properties

```diff
@@ -1,6 +1,7 @@
 # this file is used by .ci/idea_inspection.sh and .ci/idea_inspection.bat
 # Content should be the same as in TeamCity "Include / exclude patterns:"
 idea.exclude.patterns=.idea/**;\
+              .ci/*.config\
```

Comment:
IDEA treats `.ci/openjdk-projects-to-test-on.config` and `.ci/projects-for-no-exception-javadoc.config` as Lombok (!!!) configuration files and reports violations for them.

Reply:
Right decision to exclude